common/build.gradle.kts

```diff
@@ -67,5 +67,7 @@ dependencies {
 
     androidTestImplementation(libs.bundles.androidx.test)
 
+    // This fix an issue: provided Metadata instance has version 2.1.0, while maximum supported version is 2.0.0. To support newer versions, update the kotlinx-metadata-jvm library
     lintChecks(libs.androidx.runtime.lint)
+    implementation(platform(libs.compose.bom))
```

Comment:
Without this the `androidx.compose.runtime:runtime` is not the right one and lead to the metadata issue.

Reply:
Can we make this a TODO? Eventually the versions must be OK again, so we no longer have to add all the dependencies here and define the lint version separately.

Reply:
Google will eventually break it again. So I don't think we should have a todo here I prefer to keep the explanation of why we added it because it will help if it happens again or if someone search all over Github for a fix to his issue.